apps/test-suite/tests/GoogleSignIn.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+import { GoogleSignIn } from 'expo-google-sign-in';
+
+export const name = 'GoogleSignIn';
+
+export async function test({ describe, it, jasmine, expect }) {
+  describe('GoogleSignIn', () => {
+    describe('has constants', () => {
+      expect(typeof customContactId).toBe('string');
```

Comment:
Where is this variable defined? 🤔

Reply:
😅 lol in the Contacts tests that I copy/pasted.